documentation/wiki/Binary-Log.md

```diff
@@ -66,12 +66,7 @@ https://msbuildlog.com/
 
 # Collecting binary logs from Visual Studio builds
 
-If you need to capture a binary log in Visual Studio, instead of the command line, you'll need a Visual Studio plugin:
```

Comment:
It looks like this never resurfaced in Providing-Binary-Logs.md? Was that intentional? As far as I know, this still works; it's just harder.

Reply:
This was intentional. 

The info is in the project system docs that the link eventualy leads to. So this way we do not need to make sure to update the info with each release of VS and mainly we switch the order of suggestions - promoting the `MSBUILDDEBUGENGINE` as preferable option.